conans/client/packager.py

```diff
@@ -40,18 +40,26 @@ def new_method(pattern, src=""):
         conanfile.source_folder = source_folder
         conanfile.build_folder = build_folder
 
+        def recipe_hasmethod(conanfile, method):
+            if method in conanfile.__class__.__dict__:
```

Comment:
I know it is an oversimplification, but I would keep it ``return method in conanfile.__.....``

Reply:
I think you have to check if the param `method` is actually a mehotd and not an attribute. We would have to change the name of that function in that case or it could be misleading

Reply:
If it is not a method, it would have already failed, before this point. Change the name to ``recipe_has`` if you prefer